CHANGES/6800.bugfix.rst

```diff
@@ -0,0 +1 @@
+Modified websocket :py:exc:`~aiohttp.client_ws.ClientWebSocketResponse.receive_str`, :py:exc:`~aiohttp.client_ws.ClientWebSocketResponse.receive_bytes`, :py:exc:`~aiohttp.web_ws.WebSocketResponse.receive_str` & :py:exc:`~aiohttp.web_ws.WebSocketResponse.receive_bytes` methods to raise new :py:exc:`~aiohttp.WSMessageTypeError` exception, instead of generic :py:exc:`TypeError`, when websocket messages of incorrect types are received -- by :user:`ara-25`.
```

Comment:
The ones that are methods and not exceptions should use meth instead of exc

Reply:
Apologies for this. Corrected.